tasks/functions/functions.js

```diff
@@ -15,31 +15,48 @@ function add(a, b) {
  * }
  */
 function getFullName(object) {
-
+    return object.firstName + ' ' + object.lastName;
 }
 
+
 /**
  * write fuction that checks is number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+    let x = n % 2;
+    if (x == 0) {
+        return false;
+    } else {
+        return true;
+    }
 }
 
 /**
  * write function that return shortest of words in given array
  * e.g ["one", "two", "three"] should return one
  */
 function getShortest(wordArray) {
-
+    let shortestWord = wordArray[0];
+    for (const element of wordArray) {
+        if (element.length < shortestWord.length) {
+            shortestWord = element;
+        }
+    }
+
+    return shortestWord;
 }
 
 /**
  * write function that returns word google with given numbers of "o" symbols
  * e.g getGoogle(5) should return "gooooogle"
  */
 function getGoogle(n) {
-
+    let google = "g";
+    for (let j = 1; j <= n; j++) {
```

Comment:
it should have been done using repeat() function :)

return 'g' + 'o'.repeat(n) + 'gle'

Reply:
thanks, fixed